YoCode/appsettings.json

```diff
@@ -0,0 +1,5 @@
+﻿{
```

Comment:
This has sneaked back in. It should be removed from the repo - I thought it was in gitignore?

Reply:
There was an idea that appsettings is uploaded to github repo and then added to gitignore. Because this file is necessary for program to function properly, if someone clones git repo right now, that repo will not contain appsettings, thus, program will fail to run.